.github/workflows/helm-community-test.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Helm (Community-unit-test)
```

Comment:
The workflow is not community specific. It happened to live in the community workflow before but it actually applies to enterprise as well.
Remove community mentions and rename workflow file